daemon/logger/logger.go

```diff
@@ -15,4 +21,6 @@ type Logger interface {
 	Log(*Message) error
 	Name() string
 	Close() error
+	GetReader() (io.Reader, error)
+	LogPath() string
```

Comment:
@LK4D4 applied the short term fix and repushed. `startLogging` method now asserts if it's `*jsonfilelog.JSONFileLogger`.